rust/src/cliwrap/kernel_install.rs

```diff
@@ -0,0 +1,73 @@
+// If not running on container continue the current path.
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+use anyhow::{anyhow, Result};
+use camino::Utf8Path;
+use cap_std::fs::FileType;
+use cap_std::fs_utf8::Dir as Utf8Dir;
+use cap_std_ext::cap_std;
+use std::process::Command;
+
+/// Primary entrypoint to running our wrapped `kernel-install` handling.
+pub(crate) fn main(argv: &[&str]) -> Result<()> {
+    let mut arg = false;
+    if let Some(k) = argv.get(0) {
+        arg = k.eq(&"add");
+    }
```

Comment:
I think `is_install` or so would be clearer...how about:

`let is_install = matches!(argv.get(0), Some("add"));`